lib/notion_to_md/blocks/types.rb

```diff
@@ -108,34 +108,19 @@ def table_row(block)
         end
 
         def toggle(block)
-          result = []
-          result << "> [!CAUTION]"
-          result << "> **Unsupported Toggle Block**"
-          result << "> <br />"
-          result << "> Content needs to be imported manually."
-          result << "> <br />"
-          result << "> <br />"
-          result << "> This is the toggle title to help you find it in the original Notion page:"
-          result << "> <br />"
-          title = block[:rich_text].map {|text| Text.send(text[:type], text)}.join
-          result << "> **#{title}**"
-          result << "> <br />"
-          result.join("\n")
+          result = <<-TEXT
+            <details>
+              <summary>#{block.block.toggle["rich_text"].map { |text| Text.send(text[:type], text) }.join}</summary>
+              #{block.children.map(&:to_md).join}
```

Comment:
WDYT ❓ 
```suggestion
              #{block.children.map(&:to_md).join("\n")}
```

Reply:
We often have lists in the toggles so this might preserve some readability 👍